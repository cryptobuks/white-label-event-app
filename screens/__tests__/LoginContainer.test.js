import React from 'react';
import renderer from 'react-test-renderer';
import { Provider } from 'unstated';
import { LoginContainer } from '../';

describe('LoginContainer', () => {
  it('renders correctly', () => {
    const comp = (
      <Provider inject={[]}>
        <LoginContainer
          screenProps={{
            handleStorybookGesture: () => {},
            onChangeSubscription: () => {},
            userId: 1,
          }}
          navigation={{}}
        />
      </Provider>
    );
    const tree = renderer.create(comp).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
