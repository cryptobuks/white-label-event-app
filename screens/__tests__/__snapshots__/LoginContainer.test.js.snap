// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoginContainer renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "black",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "color": "white",
        "padding": 60,
        "paddingTop": 0,
      }
    }
  >
    Please login to subscribe for talks & sessions troughout the day.
  </Text>
  <View
    style={
      Object {
        "height": 100,
        "justifyContent": "space-between",
      }
    }
  >
    <View
      accessibilityComponentType={undefined}
      accessibilityLabel={undefined}
      accessibilityTraits={undefined}
      accessible={true}
      hasTVPreferredFocus={undefined}
      hitSlop={undefined}
      isTVSelectable={true}
      nativeID={undefined}
      onLayout={undefined}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={null}
      testID={undefined}
      tvParallaxProperties={undefined}
    >
      <View
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "#a0a0a0",
            "height": 40,
            "justifyContent": "center",
            "padding": 16,
          }
        }
      >
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Object {
              "color": "white",
            }
          }
        >
          Login with Facebook
        </Text>
      </View>
    </View>
    <View
      accessibilityComponentType={undefined}
      accessibilityLabel={undefined}
      accessibilityTraits={undefined}
      accessible={true}
      hasTVPreferredFocus={undefined}
      hitSlop={undefined}
      isTVSelectable={true}
      nativeID={undefined}
      onLayout={undefined}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={null}
      testID={undefined}
      tvParallaxProperties={undefined}
    >
      <View
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "#a0a0a0",
            "height": 40,
            "justifyContent": "center",
            "padding": 16,
          }
        }
      >
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Object {
              "color": "white",
            }
          }
        >
          Login with Google
        </Text>
      </View>
    </View>
  </View>
</View>
`;
